nncf/utils.py

```diff
@@ -146,6 +147,12 @@ def get_state_dict_names_with_modules(model, str_types=None, prefix=''):
             found.update(sub_found)
     return found
 
+def get_nncf_module_names_for_algorithm(algorithm_name=None):
```

Comment:
Type hints are missing. Should we really use names (strings) instead of class objects themselves?

Reply:
In my mind, for custom layers using string names is easier to use. Using classes looks weird  because you should import all classes of algorithms to ignore them.